docs/services/navigation/_index.md

```diff
@@ -115,6 +119,54 @@ The following attributes are available for `Navigation` services:
 | `meters_per_sec` | float | Optional | The default linear velocity for the [base](/components/base/) in meters per second. <br> Default: `0.3` |
 | `obstacles` | obj | Optional | Any obstacles you wish to add to the robot's path. See the [Motion Service](/services/motion/) for more information. |
 
+### Configure and calibrate the Frame System Service for GPS Navigation
+
+{{% alert title="Info" color="info" %}}
+
+The [Frame System Service](/services/frame-system/) is an internally managed and mostly static system for storing the "reference frame" of each component of a robot within a coordinate system configured by the user.
+
+It stores the required contextual information for Viam's services like [Motion](/services/motion/) and [Vision](/services/vision/) to use the position and orientation readings returned by components like [movement sensors](/components/movement-sensor/).
```

Comment:
I would like to make this sentence more precise but am not sure of the full scope of the frame service within the platform/all the services?

Reply:
Ok to remove if both of y'all think this clutters up the page! I felt drawn to add the tip in because the frame system is one of the harder parts of the platform to grasp at first, so if someone hadn't read through those docs that was reading through this doc, they might appreciate the info.